source/beamline/AMSlits.cpp

```diff
@@ -181,16 +181,22 @@ AMControl* AMSlits::horizontalCenter() const
 	return horizontalSlit_->center();
 }
 
-void AMSlits::addChildControl(AMControl *control)
+bool AMSlits::addChildControl(AMControl *control)
 {
-	if (allControls_->addControl(control))
+    if (allControls_->addControl(control)){
```

Comment:
@helfrij How come the AMSlits class maintains its own list of subcontrols, rather than using the usual mechanism provided by AMControl? Is it to listen for the connected state changed, and if so is there a possible refactor here for altering the slits to inherit from AMConnectedControl?